doc/en/source/onepage.rst

```diff
@@ -95,15 +95,29 @@ You need to change the following IP addresses in config.json according to syour
 
 .. important:: 
 
-	please DO NOT use "127.0.0.1" as the IP address of **my_hostip**, because it is only accessible to a 
+	Please DO NOT use "127.0.0.1" as the IP address of **my_hostip**, because it is only accessible to a 
 	running task inside a docker container. 
 	
-	**Firewall rules:** to make your FogFlow web portal accessible, the following ports 80 and 5672 over TCP must be open. 
+	**Firewall rules:** To make FogFlow web portal accessible and for its proper functioning, the following ports must be free and open over TCP in host machine. 
+	
+	.. code-block:: console
+
+		Component		Port
+
+		Discovery		8090 
+		Broker			8070
+		Dgraph			9082
+		Designer		8080
+		Nginx			80
+		Rabbitmq		5672
+		Worker			launches task instance over any port internally
+
+
```

Comment:
task can be launched on any free port so all the ports should not be under firewall rule.